composer.json

```diff
@@ -17,10 +17,10 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/config": "^3.4|^4.0|^5.0",
-        "symfony/dependency-injection": "^3.4|^4.0|^5.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0",
-        "symfony/mercure": "^0.2"
+        "symfony/config": "^4.3|^5.0",
```

Comment:
`symfony/*` v3.4 is still active, why removing its compatibility?

Reply:
Because we use 4.0 only features.

Reply:
So this version has a big BC break :disappointed: !

What are the 4.0 only features used in this PR?

Reply:
Isn't it possible to handle the `setDeprecated` method differently, as it's the only 4.0 feature added in this PR?

Reply:
Bumping deps isn't a BC break, but I'm not against supporting 3.4.